api/controllers/UserController.js

```diff
@@ -49,6 +51,44 @@ module.exports = {
 
   },
 
+  sendInvite: function (req, res) {
+
+    User.findOne(req.param('id')).exec(function (err, user) {
+      if (err) return next(err);
+
+      if (user) {
```

Comment:
Let's "fail quick" here and flip this condition so the rest of the route code doesn't need to be nested. So `if (!user)` return either a forbidden or server error immediately.